app/views/trends/show.html.erb

```diff
@@ -22,3 +20,4 @@
     </tbody>
   <% end %>
 </table>
+<%= add_to_dashboard :template => 'trends/trend_chart', :name => _("Trends for %s") % trend_title(@trend), :sizex=>5, :sizey => 1, :data =>{:trend_id => @trend.id, :range => range} %>
```

Comment:
Where's this `:name` used?  I'm a bit concerned about what looks like concatenation of two strings for translation.